app/views/admin/loans/show.html.slim

```diff
@@ -36,6 +34,12 @@ section.loan.details class=(@loan.valid? ? 'show-view' : 'edit-view')
 
   div.loan-content
     = render "admin/loans/tabs/tabs"
-
-  javascript:
-    $(function() { new MS.Views.ShowEditView({ el: '.loan.details' }) });
+    
+javascript:
+  $(function(){
+    new MS.Views.ShowEditView({ el: '.loan.details' })
+    new MS.Views.LoanTabView({
+      el: '.nav-tabs',
+      calEvents: #{@calEvents.to_json.html_safe}
```

Comment:
This is one of those cases where html_safe should not be used since the calEvents contain user input. This will get changed as part of the refactor anyway, so no need to touch this for now, but please keep in mind for future.